wordpress-mobile/WordPress-FluxC-Android
# Filter complex metadata

Summary
==========
Fix issue #https://github.com/woocommerce/woocommerce-android/issues/6938 by creating a set of rules and regex pattern matching to remove or adapt any metadata from a given order.

How to Test
==========
1. Fetch an order that contains multiple types of metadata values and keys, like empty values, internal keys, HTML, and JSON content.
2. Verify that HTML values get all tags removed and the text is kept, and metadata with internal keys, empty or JSON values are all removed.

Update release notes:
